apps/angular/ionic-angular-standalone/src/app/issue/not-import-one/not-import-one.page.html

```diff
@@ -0,0 +1,22 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-buttons slot="start" *ngIf="!isForce"><!-- I have checked to the point where ngIf is having an impact.-->
```

Comment:
angular control syntax affects. ion-buttons, ion-icon, and ion-spinner are not imported during migrate.